code/auth-service/src/controllers/authorisation.js

```diff
@@ -8,7 +8,7 @@ import getRoles from '../auth/authzApi';
 import getPermissions from '../permissions/permissions';
 
 const PRIVATE_KEY = fs.readFileSync(config.get('privateKey'));
-const PUBLIC_KEY = fs.readFileSync(config.get('publicKey'));
+export const PUBLIC_KEY = fs.readFileSync(config.get('publicKey'));
```

Comment:
This should be extracted into a separate file too along with the duplicated values.